03LessonThree/LessonThree.js

```diff
@@ -1,34 +1,36 @@
 module.exports = function() {
 
+
   this.buildArray = function (first, second, third) {
     // return array with the items first, second, and third in an array.
-
+    return [first, second, third];
   }
 
   this.returnThirdItem = function (arr) {
     // should return the third item in the array
-
+    return arr[2];
   }
 
   this.setFirstItem = function (arr, newFirstItem) {
     // should set the first item in the array with newFirstItem
-
+    arr[0] = (newFirstItem);
+    return arr;
   }
 
   this.returnCenterItem = function(fiveByFiveArray) {
     // returns the "center" item in a 5 x 5 array
-    // ex. 3 x 3 array [[1, 2, 3], [4, 5, 6], [7, 8, 9]] the center item is 5
-
+    // ex. 3 x 3 array [[1, 2, 3,4,5], [1, 2, 3,4,5],[1, 2, 3,4,5],[1, 2, 3,4,5],[1, 2, 3,4,5]] the center item is 5
+    return fiveByFiveArray[3],[3];
```

Comment:
This is honestly the most interesting false-positive for a test I've seen. @mistakevin 

So

```js
return fiveByFiveArray[3],[3];
```

isn't the same as
```js
return fiveByFiveArray[3][3]
```

Note the commas present in the current return statement. In JS, if you have a return statement of the form

```js
return x,y
```

It actually returns `y`.

So this `returnCenterItem` method is always returning `[3]`. It's not returning the element at row 3 column 3 in the array `fiveByFiveArray`. It's literally just returning that thing after the comma, `[3]` itself.

The [test that was written](https://github.com/AustinCodingAcademy/HTMLIntermediateSection2Tests/blob/1555141a063a210784e33814998f5d5f7e31f149/03LessonThree/test/LessonThreeTest.js#L30-L37) is making the following comparison:

```js
var arr = [
  [1, 2, 1, 4, 5],
  [1, 2, 2, 4, 5],
  [1, 2, 3, 4, 5],
  [1, 2, 4, 4, 5],
  [1, 2, 5, 4, 5]
];
assert.equal(lessonThree.returnCenterItem(arr), 3);
```

And it just so happens that, for whatever reason, this is the case in JS

```js
> [3] == 3
true
```
So @JaceDeloney , this test is passing, but it shouldn't be. Correcting the syntax here to be

```js
return fiveByFiveArray[3][3]; // no comma
```
Will show you have selected the incorrect value in the array.